Source/AnimalGenetics.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using AnimalGenetics;
 using Math = System.Math;
+using Multiplayer.API;

```

Comment:
Is this assembly part of Rimworld? If not, doing this will break builds of this mod.

I believe Rimworld has a deterministic RNG that should be used instead?

Reply:
@gregorycu this is the way native compatibility is typically added, we can try getting rid of the API and just pushing seeds to Verse.Rand regardless of if they're actually in Multiplayer or not and that might also work

With Verse.Rand we do still have to push a seed in for it to not generate different random values for each connected client